### DISCUSSION TITLE
Ambient brightness

### DISCUSSION THREAD

**Discussion by RikPeeters** [T-Troll/alienfx-tools#33] [Ideas]
can we have a 'negative' dimming option?
most of the time I can barely see the lights, even on a very bright screen the keyboard stays shallow.

**Comment by T-Troll** [comment]
I meet this effect too, but in different condition.
Seems like there are hardware dimming as well (but i didn't catch the command for it).

As a quick fix, run AWCC and disable dimming and battery dimming into it. It should help.

PS: Unfortunately, i can't check it right now - my 13R2 broken recently (battery blow and damage shell and touchpad). So we have 2 options - wait until i got spare parts and repair, or you can try to catch dimming command from AWCC using USBPcap and share it to me. I check before - AWCC just decrease brightness at dim, but seems like it's ALSO other way.

**Reply by T-Troll** [reply]
My friend, i need you help.
I'm working on fan/power control now, and need more data.
Can you please install/run `rweverything`, open "ACPI" page, save all data and send it to me?
I want to check you structures and how far it from nextgen ones.
T.I.A.

**Reply by RikPeeters** [reply]
[AcpiTbls.zip](https://github.com/T-Troll/alienfx-tools/files/6911171/AcpiTbls.zip)

Is this what you need?

**Comment by T-Troll** [comment]
About brightness.... It's a bug. Will be fixed in next release.
